### PR TITLE
Update file handling doc. Fixes #285

### LINKED ISSUE #285: document/extend placeholder file handling doc
origin: #229 file handling doc needs to be written.

### PR DESCRIPTION
**Pull request by rtnpro**
Fixes #285

### CODE REVIEW

File: docs/file_handling.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ## Fetch
 
-Fetching an Atomic App means to download the artifacts and sample answerfile.
+Fetching an Atomic App means to download the metadata files: artifacts and
+sample answerfile for an atomic app. By default, it downloads the metadata
```
Comment (dustymabe) [inline docs/file_handling.md]: Atomic App

File: docs/file_handling.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+files for the atomicapp to a directory of the form
```
Comment (dustymabe) [inline docs/file_handling.md]: Atomic App

File: docs/file_handling.md [hunk 3]
```diff
@@ -8,11 +13,30 @@ Image developers may run the root container and point to a Nulecule directory on
 
 ## Directories
 
-* `/tmp/<atomicapp_name>`: Host directory for temporary Nulecule files. May be overridden.
-* `/tmp/<atomicapp_name>/.workdir`: Host directory for artifact template files with variable substitution.
+* `external/`: External atomic apps, if any, for the given atomic app are
```
Comment (dustymabe) [inline docs/file_handling.md]: Might be good to move this to below the next entry and change the name to:

`/var/lib/atomicapp/<atomic app name>-<uuid>/external`

**Comment by dustymabe** [comment]
A few comments made in the code above. One general comment is to modify all `atomic app` instances to `Atomic App`.

If you update those and rebase, then feel free to merge. Good work!

**Comment by rtnpro** [comment]
Merging it.

**Pull request merged**